### ISSUE TITLE
Default URI is always accessible

### ISSUE THREAD

**Issue by dani8art** [ice-services/moleculer-web#22]
I have an alias on `moleculer-web`  but, the default URI is also accessible, then I can't expose the service action only in HTTP POST method.  For example: 

I have an actions: `test.greeter` and I want to expose it only in HTTP POST method. Then I configure my alias:

```javascript
{
    path: '/api/main'
    aliases:{
        'POST greeters': 'test.greeter.create'
    }
}
```
Now I have two URIs:

- /api/main/greeters
  - POST

- /api/main/test/greeter/create
  - GET
  - POST
  - DELETE
  - PUT
  - ...

Alias make a 'back door' open for all HTTP methods and for unauthenticated consumers. Is there any solution? can I block the default URIs?

**Comment by icebob** [comment]
Hi,
it's true, because alias is just an alias/shorthand for actions. It doesn't block anything.

Do you need a route option which blocks all URLs except the aliases?

**Comment by dani8art** [comment]
Yes, I think it will be a good solution.
It's possible to do that?

**Comment by icebob** [comment]
I will do it.

**Comment by dani8art** [comment]
thank you!

**Comment by icebob** [comment]
Is the `blockDirectCall` option name describes this feature? What do you think?

**Comment by dani8art** [comment]
Yes, I think it would be a good name, it describes the feature. 
If you allow me, I would like to propose you `mappingPolicy`. 

```
{
  mappingPolicy: 'restrict' #Only aliases will be mapped
}
...
{
  mappingPolicy: 'include' #Only [aliase] xor [direct]
}
...
{
  mappingPolicy: 'all' # [aliases] union [direct]
}
```
What do you think?

**Comment by icebob** [comment]
`mappingPolicy` is good. I like it. But I don't understand the `include` option. Do you have a use case for it?

**Comment by dani8art** [comment]
`include` or `xor` I don't know how to name this value. I want to describe the following use case:
You have 3 actions:
`books.create`
`books.get`
`books.info`
and you want to add aliases to `books.create` and `books.get`, but you want to map all of them. 
```
{
  mappingPolicy: 'include',
  aliases: {
    'GET books': 'books.get',
    'POST books': 'books.create'
  }  
}
```
And the result will be:

- /books
  - POST
  - GET

- /books/info
  - POST
  - GET
  - UPDATE
  - DELETE
  - ...

**Comment by icebob** [comment]
Thanks. I added `mappingPolicy` with `all` and `restrict`. Currently the `include` is not implemented, it is not too easy. I'm still thinking about it.

**Comment by dani8art** [comment]
Thank you very much!

**Comment by icebob** [comment]
Released in [v0.5.2](https://github.com/ice-services/moleculer-web/releases/tag/v0.5.2)
